client/components/preview/Preview.js

```diff
@@ -69,6 +77,28 @@ class Preview extends Component {
       console.log('loading used components');
       populateSavedComponents(this.props.usedComponents);
     }
+    if (this.props.usedStyles.length > 0) {
+      console.log('laoding saved styles!');
+      this.props.usedStyles.forEach(styleObj => {
+        let node = document.getElementById(styleObj.domId);
+        console.log('the node is ', node);
+        const styles = styleObj.styles;
+        console.log('loading curr style ', styles);
+        if (styles.fontSize.length)
+          node.style.fontSize = styles.fontSize + 'px';
+        if (styles.color.length) node.style.color = styles.color;
+        if (styles.borderStyle.length)
+          node.style.borderStyle = styles.borderStyle;
+        if (styles.borderWidth.length)
+          node.style.borderWidth = styles.borderWidth;
+        if (styles.borderRadius.length)
+          node.style.borderRadius = styles.borderRadius + 'px';
+        if (styles.padding.length) node.style.padding = styles.padding + 'px';
+        if (styles.backgroundColor.length)
+          node.style.backgroundColor = '#' + styles.backgroundColor;
```

Comment:
maybe there is a programatic way of doinf this for all styles[extant property] - Object.keys(styles).forEach ?